taxi/models.py

```diff
@@ -1,6 +1,8 @@
 from django.db import models
```

Comment:
i can see that you changed driver-list and driver-details files by adding "Me". But as far as I understood you didn't commit these files. could you please do it?

Reply:
Hi, sorry for the long reply.

Reply:
![image](https://user-images.githubusercontent.com/62405464/201786905-b5c84746-1105-4f81-a5b8-451eba169308.png)

Reply:
Somehow didn't see them. Sorry, probably was more tired than I thought( Everything looks good:)